spec/services/middleware_topology_service_spec.rb

```diff
@@ -17,7 +17,7 @@
     subject { middleware_topology_service.build_topology }
 
     let(:ems_hawkular) { FactoryGirl.create(:ems_hawkular) }
-    let(:middleware_server) do
+    let(:md_server) do
```

Comment:
no, it's not a good idea. 'Middleware' is a prefix / adjective, but the main information is in the 'server', 'deployment', 'datasource', etc.